### PR TITLE
docs(governance): create governance doc

### PR DESCRIPTION
**Pull request by jkarinkl**
# Description

Add first version of GOVERNANCE.md document. Describing current leadership- and decision making processes of RIOT-rs.

## Open Questions

Please check the references:
- Are they correctly issued?
- Is the template sufficiently mentioned? (I linked to the web page of the template, not to the actual template on GitHub. I've used both pages for reference)
- Are the creative commons description and links correctly mentioned? 

## Change checklist

- [x] I have followed the [Coding Conventions](../book/src/coding-conventions.md).
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+## Digital commons
+
+RIOT-rs is a digital public good, or digital common, supported by public funding. The project's outcome is [open source](https://github.com/future-proof-iot/RIOT-rs/blob/main/LICENSE) and freely available for everyone. 
+
+
+## Decision making
+
+Decisions within RIOT-rs are made with consensus, as much as possible, among team members, based on technical arguments. The Founder-leaders have override  power, for strategic steering and/or to break ties.
```
Comment (kaspar030) [inline GOVERNANCE.md:25]:
```suggestion
Decisions within RIOT-rs are made with consensus, as much as possible, among team members, based on technical arguments. The Founder-leaders have override power, for strategic steering and/or to break ties.
```

(drop extra space)

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+### Contributors
+
+Anyone can contribute to the RIOT-rs project, for example via comments on issues or pull requests, writing code or documentation etc. 
+
+
+## Getting involved
+
+Decisions about the RIOT-rs project are made in the matrix chat room and during the weekly meetings of team members. If you have questions about the direction the RIOT-rs project is taking, or want to get involved in the decision making process, you can get in contact via these channels.
```
[the remark this answers is not in the document]
Comment (kaspar030) [reply GOVERNANCE.md:51]: right, maybe just drop "of team members"?

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The RIOT-rs project is dedicated to creating a general-purpose operating system (OS) for low-power, microcontroller-based IoT devices, written from scratch in Rust, providing easy portability of application code to diverse microcontroller-based hardware. This governance document explains how the project is run.
+
+
+## Values
+
+RIOT-rs and its leadership embrace the following values:
+
+* Openness: Communication and decision-making happens in the open and is discoverable for future reference. As much as possible, all discussions and work take place in public forums and open repositories.
+
+* Fairness: All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
```
[the remark this answers is not in the document]
Comment (jkarinkl) [reply GOVERNANCE.md:13]: Good point! Refers to the contributions. 


```suggestion
* Fairness: All stakeholders have the opportunity to provide feedback and submit contributions. These contributions will be considered on their merits.
```

**Comment by jkarinkl** [comment]
Comments incorporated, ready to merge!

**Comment by kaspar030** [comment]
Great, thanks!

**Pull request merged**